src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,38 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
+# TO-DO: complete the helper function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
-    merged_arr = [0] * elements
+    # elements = len( arrA ) + len( arrB )
+    merged_arr = []
+    # merged_arr = [0] * elements
     # TO-DO
-    
+    arrA_index = 0
+    arrB_index = 0
+    while arrA_index<len(arrA) and arrB_index<len(arrB):
+        if arrA[arrA_index] < arrB[arrB_index]:
+            merged_arr.append(arrA[arrA_index])
+            arrA_index += 1
+        else:
+            merged_arr.append(arrB[arrB_index])
+            arrB_index += 1
+
+    merged_arr += arrA[arrA_index:]
+    merged_arr += arrB[arrB_index:]
```

Comment:
I really like you implementation of this. I was skeptical that it was going to work but it does! Even the model solution uses four different if checks so I thought because you were only using two, you might miss out on some edge cases. Good job.